applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -297,7 +297,8 @@ static void base_recv_cb(struct bt_bap_broadcast_sink *sink, const struct bt_bap
 			((active_stream_index == AUDIO_CH_L) ? CH_L_TAG : CH_R_TAG));
 		LOG_DBG("Waiting for syncable");
 	} else {
-		LOG_WRN("Found no suitable stream");
+		LOG_DBG("Found no suitable stream");
```

Comment:
Why was this changed to a DBG?

Reply:
Since the LE_AUDIO_EVT_NO_VALID_CFG event is now published on BIS headset as well (not only CIS gateway) we get a WRN print in streamctrl